DEVELOPING_ON_MODULAR.md

```diff
@@ -0,0 +1,93 @@
+# Developing on Modular Volttron
+
+
+The following documentation steps through the process of setting up your environment to start
+developing on modular VOLTTRON.
+
+There are requirements in order to follow this README. The following requirements should be installed system-wide, 
+so they are available to be used throughout the README.
+
+* Poetry 1.2.2
+
+## Installing poetry
+See [poetry installation](https://github.com/eclipse-volttron/volttron-core/tree/develop#poetry)
+
+
+# Development
+
+##  New Agent 
+* Please use the tool [copier-poetry-volttron-agent](https://github.com/VOLTTRON/copier-poetry-volttron-agent) to create a new agent. 
+
+## Existing Agent
+* Fork the Agent repo on Github
+* Clone the repo in your local environment
+* Setup the environment using poetry
+
+```shell
+poetry install 
+```
+
+* Create a working branch off of 'develop'
+* Make a code change
+* Run tests
+
+```shell
+poetry run pytest tests/
```

Comment:
shouldn't need to add tests folder

Reply:
This command doesn't add a tests folder. It runs the tests in the project using `pytest`.

Reply:
I am saying there is no need to specify tests

pytest will find them

Reply:
poetry run pytest tests/ is the same as poetry run pytest

Reply:
I see. I added the `tests/` parameter to make it for explicit, especially to readers who aren't so familiar with pytest.

Reply:
that works